### PR TITLE
Remove unused OAuth config

### LINKED ISSUE #407: [Bug]: Oauth Config still there
### Expected behavior

leftover of https://github.com/pimcore/customer-data-framework/pull/359

see https://github.com/pimcore/customer-data-framework/blob/4.x/src/DependencyInjection/Configuration.php#L36-L41

maybe there are also other places?

### Actual behavior

.

### Steps to reproduce

.

### PR DESCRIPTION
**Pull request by aryaantony92**
Resolves #407

### CODE REVIEW

**Comment by dvesh3** [comment]
@aryaantony92 please rebase to fix the tests. thanks!

File: .github/ci/files/var/classes/DataObject/SsoIdentity.php [hunk 1]
```diff
@@ -1,187 +0,0 @@
...
-use Pimcore\Model\DataObject\PreGetValueHookInterface;
-
-/**
-* @method static \Pimcore\Model\DataObject\SsoIdentity\Listing getList()
-* @method static \Pimcore\Model\DataObject\SsoIdentity\Listing|\Pimcore\Model\DataObject\SsoIdentity|null getByProvider($value, $limit = 0, $offset = 0, $objectTypes = null)
-* @method static \Pimcore\Model\DataObject\SsoIdentity\Listing|\Pimcore\Model\DataObject\SsoIdentity|null getByIdentifier($value, $limit = 0, $offset = 0, $objectTypes = null)
-* @method static \Pimcore\Model\DataObject\SsoIdentity\Listing|\Pimcore\Model\DataObject\SsoIdentity|null getByProfileData($value, $limit = 0, $offset = 0, $objectTypes = null)
-*/
-
-class SsoIdentity extends \CustomerManagementFrameworkBundle\Model\AbstractSsoIdentity
```
Comment (dvesh3) [inline .github/ci/files/var/classes/DataObject/SsoIdentity.php:27]: Remove references from other classes: https://github.com/pimcore/customer-data-framework/blob/f24b239e62717fdd4a1eb1d8f55c26b39414e0e4/install/class_source/optional/class_Customer_export.json#L1664

Comment (dvesh3) [reply .github/ci/files/var/classes/DataObject/SsoIdentity.php:27]: Remove `AbstractSsoIdentity`, `SsoIdentityServiceInterface` & `SsoIdentityInterface` as well.

File: src/DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -33,13 +33,6 @@ public function getConfigTreeBuilder()//: TreeBuilder
         $rootNode = $treeBuilder->getRootNode();
         $rootNode->addDefaultsIfNotSet();
 
-        $rootNode
-            ->children()
-                ->arrayNode('oauth_client')
```
Comment (dvesh3) [inline src/DependencyInjection/Configuration.php:38]: Adapt upgrade notes.

**Comment by aryaantony92** [comment]
Tests will be fixed by https://github.com/pimcore/customer-data-framework/pull/421

**Comment by dvesh3** [comment]
please rebase to fix tests. thanks!

File: install/class_source/optional/class_Customer_export.json [hunk 3]
```diff
@@ -1647,37 +1647,6 @@
...
-                                        "allowToCreateNewObject": true,
-                                        "optimizedAdminLoading": false,
-                                        "visibleFieldDefinitions": [],
-                                        "classes": [
-                                            {
-                                                "classes": "SsoIdentity"
-                                            }
-                                        ],
-                                        "pathFormatterClass": "",
-                                        "name": "ssoIdentities",
```
Comment (dvesh3) [inline install/class_source/optional/class_Customer_export.json:1668]: Please add to upgrade notes that field should removed and `SsoIdentity` class.

**Pull request merged**